samples/snippets/subscriber_test.py

```diff
@@ -307,7 +354,7 @@ def test_receive_with_delivery_attempts(
     # We keep retrying up to 10 minutes for mitigating the flakiness.
     @typed_backoff
     def run_sample() -> None:
-        _publish_messages(publisher_client, topic)
+        _ = _publish_messages(publisher_client, topic)
```

Comment:
Does the return value need to be stored in `_`?

Reply:
The returned messages IDs are only needed by one test. I changed them all because I added a return for the publish function.  `_` means in those other tests we don't care about the message IDs.